js.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    const body = document.querySelector('body');
+    const storage = window.sessionStorage;
+
+    let counter = storage.getItem('counter') || 0;
+
+    body.addEventListener('click', bodyClicked);
+
+    function bodyClicked() {
+      counter ++;
+      updateCounter();
+    }
+
+    function updateCounter () {
```

Comment:
I'd merge this function and bodyClicked into one. They are too simple for separate functions.